benchmarks/quant-hela-method-comparison/scripts/02_prepare_peptides.py

```
@@ -0,0 +1,513 @@
+    # Handle MSstats format - create SampleID from BioReplicate
+    if data_format == "msstats":
+        if "SampleID" not in df.columns:
+            if "BioReplicate" in df.columns:
+                df["SampleID"] = df["BioReplicate"].astype(str)
+            elif "Run" in df.columns:
+                df["SampleID"] = df["Run"].astype(str)
+        if "NormIntensity" not in df.columns and "Intensity" in df.columns:
+            df["NormIntensity"] = df["Intensity"]
+
+    # Target columns we need
+    targets = ["ProteinName", "PeptideSequence", "SampleID", "NormIntensity"]
```

Ruff (0.14.14)

[error] Local variable `targets` is assigned to but never used

Remove assignment to unused variable `targets`

(F841)